src/main/java/org/breedinginsight/brapi/v2/BrAPIV2Controller.java

```diff
@@ -186,6 +187,15 @@ public HttpResponse<?> getCatchall(@PathVariable("path") String path, @PathVaria
         return executeRequest(path, programId, request, "GET");
     }
 
+    @Delete("/${micronaut.bi.api.version}/programs/{programId}" + BrapiVersion.BRAPI_V2 + "/{+path}")
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
As we discussed in standup, this currently won't delete the associated dataset and that should be captured in the the experiment delete endpoint implementation.